spec/config-spec.js

```diff
@@ -1,55 +1,46 @@
-/*
- * decaffeinate suggestions:
- * DS101: Remove unnecessary use of Array.from
- * DS102: Remove unnecessary code created because of implicit returns
- * Full docs: https://github.com/decaffeinate/decaffeinate/blob/main/docs/suggestions.md
- */
 const path = require('path');
 const fs = require('fs-plus');
 const temp = require('temp');
 const apm = require('../lib/apm-cli');
 
-describe("apm config", function() {
-  let [atomHome, userConfigPath] = Array.from([]);
+describe('apm config', () => {
+  let userConfigPath;
 
-  beforeEach(function() {
+  beforeEach(() => {
     spyOnToken();
     silenceOutput();
-
-    atomHome = temp.mkdirSync('apm-home-dir-');
+    const atomHome = temp.mkdirSync('apm-home-dir-');
     process.env.ATOM_HOME = atomHome;
     userConfigPath = path.join(atomHome, '.apmrc');
-
-    // Make sure the cache used is the one for the test env
-    return delete process.env.npm_config_cache;
+    delete process.env.npm_config_cache;
```

Comment:
```suggestion
    // Make sure the cache used is the one for the test env
    delete process.env.npm_config_cache;
```

Probably best to leave any comments around for the next person. Just in case